app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    file_name, new_path = command.split(" ")[1:]
+
+    with open(file_name, "r") as sorce:
+        text = sorce.read()
+
+    os.remove(file_name)
```

Comment:
The `os.remove(file_name)` line will raise an error if the file does not exist or if there are permission issues. Consider handling potential exceptions here as well.